vcpkg.json

```diff
@@ -2,7 +2,8 @@
   "builtin-baseline": "01f602195983451bc83e72f4214af2cbc495aa94",
   "dependencies": [
     "sqlite3",
-    "woff2"
+    "woff2",
+    "fontconfig"
```

Comment:
I think we want this to look more like this:

```
 "dependencies": [
    "sqlite3",
    "woff2",
    {
        "name": "fontconfig"
        "platform": "linux | openbsd | freebsd"
    }
],
```
https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#platform-expression

Reply:
or, perhaps we don't want fontconfig from vcpkg at all, and only want to use it if it's installed on the target system? that would mean the check in your cmake file should have `QUIET` instead of `REQUIRED`

```
find_package(Fontconfig CONFIG QUIET)
if (Fontconfig_FOUND)
    set(HAS_FONTCONFIG ON CACHE BOOL "" FORCE)
    add_compile_definitions(USE_FONTCONFIG=1)
endif()
```

Reply:
We'll definitely need fontconfig, since that's how we'll stop loading every font ever in favor of only loading what's used :sweat_smile: